### PR TITLE
Blocking Either as IO in BlockingUtils

### PR DESCRIPTION
**Pull request by wodomierz**

### CODE REVIEW

File: commons-core/jvm/src/main/scala/com/avsystem/commons/concurrent/BlockingUtils.scala [hunk 1]
```diff
@@ -38,6 +39,10 @@ abstract class BlockingUtils {
   def asTask[T](blockingCode: => T): Task[T] =
     Task.eval(blockingCode).executeOn(ioScheduler, forceAsync = true)
 
+  def asIO[A, B](blockingCode: => Either[A, B]): IO[A, B] = IO.deferTotal {
```
Comment (ghik) [inline commons-core/jvm/src/main/scala/com/avsystem/commons/concurrent/BlockingUtils.scala:42]: Can you please debrace this into one line?

File: commons-core/jvm/src/main/scala/com/avsystem/commons/concurrent/BlockingUtils.scala [hunk 2]
```diff
@@ -38,6 +39,10 @@ abstract class BlockingUtils {
...
+    IO.fromEither(blockingCode)
+  }.executeOn(ioScheduler, forceAsync = true)
```
Comment (ghik) [inline commons-core/jvm/src/main/scala/com/avsystem/commons/concurrent/BlockingUtils.scala:44]: `forceAsync = true` is the default AFAIR

Comment (wodomierz) [reply commons-core/jvm/src/main/scala/com/avsystem/commons/concurrent/BlockingUtils.scala:44]: I know, I just want to be consistent with the asTask impl.

File: commons-core/jvm/src/main/scala/com/avsystem/commons/concurrent/BlockingUtils.scala (hunk 1 above)
Comment (wodomierz) [reply commons-core/jvm/src/main/scala/com/avsystem/commons/concurrent/BlockingUtils.scala]: done

**Review by ghik — APPROVED** [review]

**Pull request merged**